index.js

```diff
@@ -18,30 +19,48 @@ var startTime = function() {
   });
 };
 
+var potentiallyMigrateDatabase = function() {
```

Comment:
I feel a little weird about this function being defined directly in the integration piece of code, which (in my head) was meant to tie disparate pieces of code together. For OCD purposes, I might pull this function definition out and put it into its own file. However, I'm not sure OCD is a good reason to move things around ;)